internal/gen/controller.swagger.json

```diff
@@ -89,12 +89,14 @@
         "parameters": [
           {
             "name": "auth_method_id",
+            "description": "",
```

Comment:
These empty descriptions should be functionally equivalent to what we had before